gno.land/pkg/integration/testing_integration.go

```diff
@@ -141,29 +139,13 @@ func SetupGnolandTestScript(t *testing.T, txtarDir string) testscript.Params {
 						ts.Setenv("RPC_ADDR", laddr)
 						ts.Setenv("GNODATA", gnoDataDir)
 
-						const listenerID = "testing_listener"
-
-						// Wait for first block by waiting for `EventNewBlock` event.
-						nb := make(chan struct{}, 1)
-						node.EventSwitch().AddListener(listenerID, func(ev events.Event) {
-							if _, ok := ev.(types.EventNewBlock); ok {
-								select {
-								case nb <- struct{}{}:
-								default:
-								}
-							}
-						})
-
-						if node.BlockStore().Height() == 0 {
-							select {
-							case <-nb: // ok
-							case <-time.After(time.Second * 6):
-								ts.Fatalf("timeout while waiting for the node to start")
-							}
+						// Wait for first block.
+						select {
+						case <-time.After(time.Second * 6):
```

Comment:
What do you think about bumping this to `10s`?

The reasoning behind it is most of our consensus timeouts are ~10s, and this functionality relies on the fact the node itself is able to produce a block within that interval